src/test/kotlin/sirius/web/http/CSRFTokenTest.kt

```diff
@@ -27,7 +28,7 @@ class CSRFTokenTest {
 
         val result = TestRequest.GET("/test/fake-delete-data").execute()
 
-        result.getStatus() == HttpResponseStatus.INTERNAL_SERVER_ERROR
+        assertEquals(result.getStatus(), HttpResponseStatus.INTERNAL_SERVER_ERROR)
```

Comment:
**Optional:** Change the order of the parameters. The first parameter should contain the expected value and the second one the actual value.
This improves the clarity of the error message when a test fails.